test/typing/plain_files/orm/typed_queries.py

```diff
@@ -440,6 +440,27 @@ def t_dml_insert() -> None:
     reveal_type(r3)
 
 
```

Comment:
we should have some tests:

* `op()` change: test/typing/plain_files/sql/operators.py
* `literal` change: test/typing/plain_files/sql/common_sql_expr.py
* `of` change (Core select() version): test/typing/plain_files/sql/common_sql_expr.py , there are some with_for_update tests there already